MarsRover.Tests/OutputFormatterShould.cs

```diff
@@ -21,15 +21,15 @@ public void FormatMap_ReturnsMapWithNoRover_GivenMap(List<Square> squares, int w
         
         [Theory]
         [MemberData(nameof(OutputFormatterShouldTestData.FormatMapWithRoverTestData), MemberType = typeof(OutputFormatterShouldTestData))]
-        public void FormatMap_ReturnsMapWithRover_GivenMapAndRover(List<Square> squares, int width, int height, params string[] inputs)
+        public void FormatMap_ReturnsMapWithRover_GivenMapAndRover(List<Square> squares, int width, int height, int x, int y, Direction direction, params string[] inputs)

```

Comment:
`inputs` is misleading. perhaps a name that reflects that these are the lines that make up your expected map?